check/README.md

```diff
@@ -0,0 +1,99 @@
+# Tool for work with string tables
+
+## Glossary
+
+- `string table` a file with translation
+- `reference table` en.txt string table.
+  English is the main game language, so any changes related to text should be in that file.
+  Also this table us used as fallback for other translation.
+  This file is mostly maintained by people who change game code,
+  so the list of entries in this file should be the most actual.
+- `entry` one piece of translation, it consists of
+  - `key` a sting that represents entry in the codebase (C++, Python).
+    This string must be unique.
+  - `value` actual translation that will be shown to the user.
+
+
+### Key
+Only English chars in uppercase and dash are allowed.
+Rename of a key should be done with caution. Usually key used as is in code,
+but sometimes it could be used as prefix and code logic based on name could be present.
+
+### Value syntax
+#### Simple (single line)
+A single line of text, should not be started and ended by any whitespace character.
+
+```
+KEY
+value
+
+>>>
+value
+<<<
+```
+
+#### Full (single or multiline)
+Start and end with triple single quotes.
```

Comment:
maybe **triple** -> **three**

maybe **single quotes** -> **apostrophes**

Reply:
triple is often used in Python [documents](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings).

> For consistency, always use """triple double quotes""" around docstrings. Use r"""raw triple double quotes""" if you use any backslashes in your docstrings. For Unicode docstrings, use u"""Unicode triple-quoted strings""".

In the [Python documentation about string](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str) this is called `Triple quoted`, but I find it measleading.

Reply:
OK